README.md

```diff
@@ -3,167 +3,18 @@
 ![Linux](https://img.shields.io/badge/os-linux-green.svg?style=flat)
 
 # Getting started with Swift on IBM Cloud
-To get started, we'll take you through a sample *Getting Started* application that takes only few minutes to deploy.
+The Getting Started tutorial for Swift uses this sample application to provide you with a sample workflow for working with any Swift app; you set up a development environment, deploy an app locally and on the cloud, and then integrate an IBM Cloud database service in your app.
 
-In order to deploy the application to IBM Cloud, you'll need an [IBM Cloud account](https://console.ng.bluemix.net/registration/). Once registered, you can quickly get your app setup remotely using our deploy to IBM Cloud button. Then, simply follow the instructions [here](#5.-Add-a-database) to set up and connect your cloudant database.
+The Swift app uses the [Kitura CouchDB package](https://github.com/IBM-Swift/Kitura-CouchDB) to interact with the [Cloudant Java Client](https://github.com/cloudant/java-cloudant) to add information to a database and then return information from a database to the UI.
```

Comment:
Kitura CouchDB doesn't use the Cloudant Java Client. Could we rephrase this along the lines of what node did.

Something like...

The Swift app uses the [Kitura Web Framework](http://www.kitura.io/) and a [Cloudant NoSQL DB](https://console.bluemix.net/catalog/services/cloudant-nosql-db) to illustrate how to store information in a database and then return information from a database to the UI. To learn more about how the app connects to Cloudant, see the [Kitura CouchDB Library](https://github.com/IBM-Swift/Kitura-CouchDB)